website/docs/providers/type/community-index.html.markdown

```diff
@@ -45,16 +45,15 @@ please fill out this [community providers form](https://docs.google.com/forms/d/
 - [Cloud Foundry](https://github.com/cloudfoundry-community/terraform-provider-cf)
 - [Cloud.dk](https://github.com/danitso/terraform-provider-clouddk)
 - [Cloudability](https://github.com/skyscrapr/terraform-provider-cloudability)
-- [CloudAMQP](https://github.com/cloudamqp/terraform-provider)
+- [CloudAMQP](https://github.com/cloudamqp/terraform-provider-cloudamqp)
 - [Cloudforms](https://github.com/GSLabDev/terraform-provider-cloudforms)
-- [CloudKarafka](https://github.com/cloudkarafka/terraform-provider)
-- [CloudMQTT](https://github.com/cloudmqtt/terraform-provider)
+- [CloudKarafka](https://github.com/CloudKarafka/terraform-provider-cloudkarafka)
+- [CloudMQTT](https://github.com/CloudMQTT/terraform-provider-cloudmqtt)
 - [CloudPassage Halo](https://gitlab.com/kiwicom/terraform-provider-cphalo)
 - [CodeClimate](https://github.com/babbel/terraform-provider-codeclimate)
 - [Confidant](https://github.com/stripe/terraform-provider-confidant)
-- [Confluent Cloud](https://github.com/riferrei/terraform-provider-ccloud)
```

Comment:
Does anyone know if this is the same as https://github.com/sapcc/terraform-provider-ccloud ? The expanded names don't match ("Converged Cloud"). Well, I suppose if so they can ask to be re-added.